js/ui/iconGridLayout.js

```diff
@@ -22,6 +22,12 @@ const FOLDER_DIR_NAME = 'desktop-directories';
 const DEFAULT_CONFIGS_DIR = Config.DATADIR + '/eos-shell-content/icon-grid-defaults';
 const DEFAULT_CONFIG_NAME_BASE = 'icon-grid';
 
+const PREPEND_CONFIGS_DIR = '/var/eos-image-defaults/icon-grid';
```

Comment:
Can you make sure that this is keyed off a variable in `Config` instead of hardcoding `/var`? E.g. `Config.LOCALSTATEDIR`